src/tests/components/App.test.js

```diff
@@ -1,6 +0,0 @@
-import { render, screen } from "@testing-library/react";
-import App from "../../components/App";
-
-describe("App", () => {
```

Comment:
is this test unfinished or did you not know how to do it? If it's the latter let me know and I'll talk you through it